test/sanity/code-smell/line-endings.sh

```diff
@@ -1,13 +0,0 @@
-#!/bin/sh
-
-grep -rIPl '\r' . \
-    --exclude-dir .git \
-    --exclude-dir .tox \
-    | grep -v -F \
-    -e './test/integration/targets/win_regmerge/templates/win_line_ending.j2'
```

Comment:
Instead of removing the entire test, just add the necessary exclusions to the grep command to skip the files which legitimately need Windows line endings.